tools/kagami/src/main.rs

```diff
@@ -69,7 +69,10 @@ impl<T: Write> RunArgs<T> for Args {
 
```

Comment:
> Current and fixed version of kagami would provide different keys for the same seed.

What is the reason for this?

Reply:
Because we previously passed seed string as is to `ursa` and now we compute hash of seed string and pass this hash into `ursa`.

Reply:
Ah, I see. If `secp256k1` already have production uses, there might be problem with account recoveries or else. Just for confirmation @appetrosyan